cli/cmd/sync_v3.go

```diff
@@ -249,3 +242,28 @@ func syncConnectionV3(ctx context.Context, sourceClient *managedplugin.Client, d
 	fmt.Printf("%s. Resources: %d, Errors: %d, Warnings: %d, Time: %s\n", msg, totalResources, totals.Errors, totals.Warnings, syncTimeTook.Truncate(time.Second).String())
 	return nil
 }
+
+func tableNameFromSchema(sc *arrow.Schema) string {
+	tableName, _ := sc.Metadata().GetValue("cq:table_name")
```

Comment:
Is there a const we can use here? Maybe move `cq:*` consts to `plugin-pb-go` if we don't want to import the SDK?

Reply:
yeah we can move this later to `plugin-pb-go` just didn't want to do it now.